js-miniapp-bridge/bridge.js

```diff
@@ -0,0 +1,42 @@
+var messageQueue = []
+window.MiniAppBridge = {};
+var isPlatform = {
+    Android: function() {
+        return navigator.userAgent.match(/Android/i);
+    },
+    iOS: function() {
+        return navigator.userAgent.match(/iPhone|iPad|iPod/i);
+    }
+};
+
+MiniAppBridge.exec = function(action, onSuccess, onError) {
+    const callback = {}
+    callback.onSuccess = onSuccess;
+    callback.onError = onError;
+    callback.id = Math.random();
```

Comment:
How about using UUIDs instead?

Reply:
I don't think JavaScript has a built in method for generating a UUID. I mentioned using just a sequential ID. So we can have some sort of `next` method that returns `id++`. We just need each ID to be unique during the session.

Reply:
Yes, JS doesn't have a built-in method for generating a UUID. Updated code to have sequential unique ID now